src/collectors/diskusage/diskusage.py

```diff
@@ -63,7 +63,8 @@ def get_default_config(self):
                          '|sd[a-z]+[0-9]*$' +
                          '|x?vd[a-z]+[0-9]*$' +
                          '|disk[0-9]+$' +
-                         '|dm\-[0-9]+$'),
+                         '|dm\-[0-9]+$' +
+                         '|nvme[0-9]+[a-z]+[0-9].*$'),
```

Comment:
perhaps something like `nvme[0-9]*+[a-z]+[0-9]*([a-z]+[0-9]*)?$` would be more appropriate than a trailing wildcard match. 

it would need some testing to see if the optional match is too much of resource hit.